### PR TITLE
Add new logo to the project

### PR DESCRIPTION
**Pull request by raghu-kamath**
This Pull Request intends to add a new word-mark for the project.

The logotype includes the alphabet 'Q' written in Morse code, this was inspired by the use of morse code in the logotype of a downstream project - Kaidan.  The logotype loosely resembles a chat speech bubble, hinting that this library is used to facilitate chat communication. The morse code enclosed within the curly braces which tries to denote that this is a library meant to be used for making applications

I have included the light and dark version of the logo and also the source svg.

Adding the logo here for the conversation

qxmpp-wordmark

### CODE REVIEW

**Comment by melvo** [comment]
Thanks for opening the PR :) Could you please make sure that the graphics are [optimized](https://invent.kde.org/network/kaidan/-/wikis/optimizing-graphics)?

**Comment by raghu-kamath** [comment]
@melvo I optimised the svg files and also split them into two separate dark and light themed versions. The png files are already optimised.

**Comment by melvo** [comment]
It seems that the PNG files are not as optimized as they would be if you used https://invent.kde.org/network/kaidan/-/wikis/optimizing-graphics#portable-network-graphics-png . Could you please try that again?

**Comment by melvo** [comment]
I suggest to simply use the term `logo` instead of `word mark` and remove the prefix `qxmpp-` from the filenames.

**Comment by raghu-kamath** [comment]
Okay I will make those changes.

**Comment by raghu-kamath** [comment]
Hi @melvo 👋🏽

I optimised the png according to the instructions 
Screenshot_20230123_113306

I also changed the name of the files according to your suggestion.

**Comment by melvo** [comment]
That looks perfect! Could you also add the line
```
<schema:logo rdf:resource='https://raw.githubusercontent.com/qxmpp-project/qxmpp/master/assets/logo-light.svg'/>
```
after https://github.com/qxmpp-project/qxmpp/blob/master/doc/doap.xml#L26? That would make it possible for https://xmpp.org/software/libraries/qxmpp/ to display the new logo.

**Comment by melvo** [comment]
Please also add `.license` files (similar to https://invent.kde.org/melvo/xmpp-providers/-/blob/master/badge.svg.license or https://invent.kde.org/websites/kaidan-im/-/blob/master/images/contributing.svg.license, depending on the license you would like to use) for each new file.

@lnjX What would be your preferred license for the logo?

**Comment by raghu-kamath** [comment]
> @lnjX What would be your preferred license for the logo?

I think the logo should be trademarked so that other projects don't misuse it. And it can be kept in a cc-by-sa license so that people can share and use it.

But this is just my opinion please let me know which license to add I will add it.

**Comment by lnjX** [comment]
I prefer CC BY-SA 4.0

**Comment by melvo** [comment]
We could also add the theme-specific colors to the SVG as it is done in https://github.com/xsf/xmpp.org/commit/c00e41c286d3ed820dc1dfdaf6d5cf569eb3baeb for the XMPP logo of https://xmpp.org.

@raghu-kamath @lnjX What do you think about it?

**Comment by raghu-kamath** [comment]
I think it makes sense and will not need the png files.  do all the places where this will be shown will have support for svg?

**Comment by raghu-kamath** [comment]
I removed all the complexities of multiple images and svg files. Now it is only one svg and there is a css style inside the svg which will make it switch to light and dark. We can use this logo multiple places like the https://xmpp.org/software/libraries/qxmpp/ page and it will also adapt to light and dark mode on that site too if the same sgv is linked.

File: README.md [hunk 1]
```diff
@@ -3,14 +3,16 @@ SPDX-FileCopyrightText: 2009 Manjeet Dahiya <manjeetdahiya@gmail.com>
 
 SPDX-License-Identifier: CC0-1.0
 -->
-
 <h1 align="center">
-    QXmpp
+<picture>
+    <img alt="qxmpp" src="./logo.svg" height="150">
```
Comment (melvo) [inline README.md:8]:
```suggestion
    <img alt="QXmpp logo" src="./logo.svg" height="150">
```

**Comment by melvo** [comment]
@raghu-kamath It looks fine! Could you only optimize the SVG again and remove empty spaces and line breaks?

**Comment by melvo** [comment]
If you are done, you can squash all commits and use a commit message like *Add new QXmpp logo*.

**Comment by raghu-kamath** [comment]
@lnjX @melvo Is there any other feedback on this PR?

**Comment by melvo** [comment]
@raghu-kamath I am fine with the current state. @lnjX has to review and accept it now.

**Review by lnjX — APPROVED** [review]

**Pull request merged**